plugin/Controller/AdminViewController.php

```diff
@@ -311,17 +311,15 @@ public function enqueue_css()
 
 	public function enqueueExtraJs($hook)
 	{
-		if (__String::getNew($hook)->contains('onoffice')) {
```

Comment:
Removing the if condition here will result in `$pObject->doExtraEnqueues();` will be called on every page.

Reply:
That's right, cause the warning is appeared on every pages.